ui/src/environments/environment.local.ts

```diff
@@ -15,6 +15,7 @@ export const environment: Environment = {
   gaUserAgentDimension: 'dimension1',
   gaLoggedInDimension: 'dimension2',
   zendeskHelpCenterUrl: 'https://aousupporthelp1580753096.zendesk.com/hc',
+  createBillingAccountHelpUrl: 'https://aousupporthelp1580753096.zendesk.com/hc/en-us/articles/360039550031-Instructions-to-Create-a-Billing-Account',
```

Comment:
Fair amount of duplication here, WDYT about just including the article ID here and then combining the rest in code?

Reply:
I was thinking about it but in the end decided that the work required was about the same either way and I didn't want to constrain myself to an abstraction I'm not sure about.

Duplication here doesn't seem too bad to me. I'm guessing the article IDs will have to change anyways if we change the base url.